bootstrap.d/app-shells.yml

```diff
@@ -48,3 +48,37 @@ packages:
       - args: ['cp', '@SOURCE_ROOT@/extrafiles/profile', '@THIS_COLLECT_DIR@/etc']
       - args: ['cp', '@SOURCE_ROOT@/extrafiles/bash.bashrc', '@THIS_COLLECT_DIR@/etc']
       - args: ['ln', '-sf', 'bash', '@THIS_COLLECT_DIR@/usr/bin/sh']
+
+  - name: fish
+    metadata:
+      summary: Friendly interactive SHell
+      description: Smart and user-friendly portable command line shell.
+      spdx: 'GPL-2'
```

Comment:
`GPL-2` is not the correct SPDX for `fish`. It should be `GPL-2.0-only`. [This](https://spdx.org/licenses/) is an amazing list for SPDX identifiers.